### ISSUE TITLE
[Proposal 0005] Allow labellers to have custom report flows

### ISSUE THREAD

**Issue by heikadog** [bluesky-social/proposals#63]
It's been occasionally mentioned in Ozone issues like [this](https://github.com/bluesky-social/ozone/issues/121), but I feel like this proposal would be much more successful if moderation labellers could have their own report flows. (e.g. someone could report to a moderation labeller for transphobia or something.) From talking to several people, I've found that people tend to accidentally send reports to my moderation labeller because the report flow looks the same for everything, including the Bluesky moderation service. [Anti-Alf Aktion even got a report for CSAM](https://bsky.app/profile/scoiattolo.mountainherder.xyz/post/3l4r4xr5av22y), so at this point, with how many users are on the site now, it could potentially save people from trauma and get more people to understand and use labellers.

**Comment by bnewbold** [comment]
Yup! This is definitely on our radar, as part of an iteration on reporting flows. Labelers should be able to indicate what type of reports they can handle (either linked to specific label values, or as broad universal categories), and also able to opt out of receiving reports in general. There are also some design decisions around whether to support fanning out reports to multiple services in a single submission, or to require sending individual reports to individual services manually.
